l10n_th_asset_register_report/data/menu_report.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <menuitem
+        parent="account.menu_finance_reports"
+        id="menu_asset_register_reports"
+        name="Asset Reports"
+        groups="account.group_account_manager,account.group_account_user"
+        />
+
+    <menuitem
+        parent="menu_asset_register_reports"
+        action="action_asset_register_wizard"
+        id="menu_asset_register_report_wizard"
+        sequence="10"
+        />
+
+    <!-- PDF/HMTL REPORTS -->
+    <report
+      id="action_report_asset_register_qweb"
+      model="report_asset_register"
+      string="Asset Register"
+      report_type="qweb-pdf"
+      name="l10n_th_asset_register_report.report_asset_register_qweb"
+      file="l10n_th_asset_register_report.report_asset_register_qweb"
+      />
+    <report
+      id="action_report_asset_register_html"
+      model="report_asset_register"
+      string="Asset Register"
+      report_type="qweb-html"
+      name="l10n_th_asset_register_report.report_asset_register_qweb"
+      file="l10n_th_asset_register_report.report_asset_register_html"
+      />
+
+    <!-- PDF REPORTS : paperformat -->
+    <record id="report_qweb_paperformat" model="report.paperformat">
+        <field name="name">Asset Register report qweb paperformat</field>
+        <field name="default" eval="True"/>
+        <field name="format">custom</field>
+        <field name="page_height">297</field>
+        <field name="page_width">210</field>
+        <field name="orientation">Portrait</field>
+        <field name="margin_top">12</field>
+        <field name="margin_bottom">8</field>
+        <field name="margin_left">5</field>
+        <field name="margin_right">5</field>
+        <field name="header_line" eval="False"/>
+        <field name="header_spacing">10</field>
+        <field name="dpi">110</field>
+    </record>
+
+    <record id="action_report_asset_register_qweb" model="ir.actions.report">
```

Comment:
Why not set paperformat_id in,
```
    <report id="action_report_asset_register_qweb"
```
If you can't, because there is no such field in `<report`, at least you should move it next to each other. You need to move the paper format above all?